pytest_catchlog.py

```diff
@@ -154,6 +175,73 @@ def pytest_runtest_teardown(self, item):
         with self._runtest_for(item, 'teardown'):
             yield
 
+    def configure_console_handler(self, verbosity):
+        # Prepare the handled_levels dictionary
+        log_levels = []
+        handled_levels = {}
+        available_levels = set([
+            #logging.CRITICAL,  # This will be the default console level if no -v is passed
+            logging.ERROR,
+            logging.WARNING,
+            logging.INFO,
+            logging.DEBUG,
+            logging.NOTSET
+        ])
+        for level in get_option_ini(self.config, 'log_extra_levels'):
+            try:
+                level_num = int(level)
+            except ValueError:
+                level_num = logging.getLevelName(level)
```

Comment:
That's true only for Python 2. Here're [docs](https://docs.python.org/3.4/library/logging.html#logging.getLevelName) for Python 3.4:

> *Changed in version 3.4:* In Python versions earlier than 3.4, this function could also be passed a text level, and would return the corresponding numeric value of the level. This undocumented behaviour was considered a mistake, and was removed in Python 3.4, but reinstated in 3.4.2 due to retain backward compatibility.